.github/workflows/backend.yml

```diff
@@ -0,0 +1,32 @@
+name: Backend Testing/Linting
+
+on:
+  push:
+    paths:
+      - 'feeder/**'
+      - 'tests/**'
+  pull_request:
+    paths:
```

Comment:
Maybe we should just test all the time. The problem is that dependencies may alter behavior. It might be better to have an exclusion like "!!docs/" (or whatever the syntax is) instead of inclusion list.

Reply:
Agreed